dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -325,7 +333,7 @@ def runValidation(self, jobId, interfaces):
 
         # Get file size and write to jobs table
         if CONFIG_BROKER["use_aws"]:
-            fileSize = s3UrlHandler.getFileSize("errors/"+jobTracker.getReportPath(jobId))
+            fileSize = s3UrlHandler.getFileSize("errors/"+errorFileName)
```

Comment:
It's not super clear, but the error and warning reports on S3 are prefixed with `errors/`. So the code that goes to S3 to get the file size needs the prefix, and the non-S3 code will just get the file name as stored in the database. (`errorFileName` doesn't do any AWS checking).

Does that make sense, or am I missing something (entirely possible, as you can see). I'll put a comment here.